### PR TITLE
Fix overflow on link dialogue

### PR DESCRIPTION
**Pull request by luisbettencourt**
## 🌟 What is the purpose of this PR?

When the link is too long it overflows outside the Link Dialogue.

image

## 🔗 Related links

- [Asana task](https://app.asana.com/0/1200211978612931/1202511994490437/f) _(internal)_

## 🔍 What does this change?

- Added `word-break: break-all` to the link container.

## 📹 Demo

image

### CODE REVIEW

**Review by nathggns — DISMISSED** [review]
This is definitely better than current situation, so thank you! We may want an ellipsis overflow though. @nonparibus should take a look.

**Comment by nonparibus** [comment]
Let's wrap to 4 lines (still a lot) and then ellipsis.

Rationale:

- most services are too eager to truncate and showing a lil more here is preferable (and up to a certain point not too overwhelming)
- as Nate flags/implies, infinite is probably not desirable either.

**Comment by luisbettencourt** [comment]
Should we limit the number of characters on the text input also, or just hide with ellipsis when the url is too long? @nonparibus @nathggns

**Comment by nonparibus** [comment]
> Should we limit the number of characters on the text input also, or just hide with ellipsis when the url is too long? @nonparibus @nathggns

No

**Comment by luisbettencourt** [comment]
> Let's wrap to 4 lines (still a lot) and then ellipsis.
> 
> Rationale:
> 
> * most services are too eager to truncate and showing a lil more here is preferable (and up to a certain point not too overwhelming)
> * as Nate flags/implies, infinite is probably not desirable either.

Done in 55bd66ffa4d880955cee8d01fdd4c9ff8a7afc7f

Also removed the list decorator that I just noticed were overflowing the popup:

image

File: packages/hash/frontend/src/blocks/page/createFormatPlugins/LinkModal.tsx [hunk 1]
```diff
@@ -69,7 +69,16 @@ export const LinkModal: FunctionComponent<LinkModalProps> = ({
               >
                 <LanguageIcon className={tw`!text-base mr-1`} />
                 <div>
-                  <p className={tw`text-sm leading-none mb-0.5`}>
+                  <p
+                    className={tw`text-sm leading-none break-all mb-0.5`}
+                    style={{
+                      display: "-webkit-box",
+                      WebkitLineClamp: 4,
+                      WebkitBoxOrient: "vertical",
```
Comment (nathggns) [inline packages/hash/frontend/src/blocks/page/createFormatPlugins/LinkModal.tsx:77]: Does this work in firefox etc?

Comment (luisbettencourt) [reply packages/hash/frontend/src/blocks/page/createFormatPlugins/LinkModal.tsx:77]: Yup, doesn't work with Internet Explorer tho.
https://developer.mozilla.org/en-US/docs/Web/CSS/-webkit-line-clamp#browser_compatibility
https://developer.mozilla.org/en-US/docs/Web/CSS/box-orient#browser_compatibility

**Review by nathggns — APPROVED** [review]

**Pull request merged**